README.md

```diff
@@ -1,7 +1,6 @@
 # Contentful Batch Libs
 
 [![npm](https://img.shields.io/npm/v/contentful-batch-libs.svg)](https://www.npmjs.com/package/contentful-batch-libs)
-[![Build Status](https://travis-ci.org/contentful/contentful-batch-libs.svg?branch=master)](https://travis-ci.org/contentful/contentful-batch-libs)
 [![Dependency Status](https://david-dm.org/contentful/contentful-batch-libs.svg)](https://david-dm.org/contentful/contentful-batch-libs)
```

Comment:
I think that badge service doesn't even exist anymore :D you can remove all except npm